### ISSUE TITLE
Confidence scores for the predictions generated?

### ISSUE THREAD

**Issue by Awaisn25** [facebookresearch/seamless_communication#358]
Is there a way to get the output logits, or somehow get word-level confidence scores for the predictions generated?
